File: test_create_session.py
import httpx
import asyncio
import json

async def main():
    async with httpx.AsyncClient() as client:
        try:
            print("Sending POST request to /api/v1/sessions...")
            resp = await client.post('http://localhost:8000/api/v1/sessions', json={})
            print("Status:", resp.status_code)
            print("Body:", json.dumps(resp.json(), indent=2))
        except Exception as e:
            print("Error:", e)

asyncio.run(main())

File: test_create_session.js
const fetch = require('node-fetch'); // or just fetch if node 18+

async function test() {
  try {
    const res = await fetch('http://localhost:8000/api/v1/sessions', {
      method: 'POST',
      headers: { 'Content-Type': 'application/json' },
      body: JSON.stringify({})
    });
    const data = await res.json();
    console.log("Status:", res.status);
    console.log("Data:", data);
  } catch (err) {
    console.error("Error:", err);
  }
}
test();